src/kirby/components/page/page.component.ts

```diff
@@ -15,6 +14,7 @@ import {
   AfterContentInit,
```

Comment:
:fire: Remove unused import
```suggestion
```

Reply:
It is not unused. Is still used on `PageActionsComponent`.